about/policy/sgid/index.md

```diff
@@ -29,41 +46,52 @@ A corresponding record to the [Data Stewardship spreadsheet](https://docs.google
+We will create a hosted feature service of the dataset in our ArcGIS Online organization. The ArcGIS Online item for this feature service will reside in the corresponding ISO category folder and contain the appropriate tags. If the dataset is authoritative, it will be marked as `authoritative`. We don't anticipate some datasets ever receiving updates, such as the boundary lines for adjacent counties in the surrounding states. These datasets will only exist in ArcGIS Online and will be tagged as `static`.
+
+The [Auditor](https://github.com/agrc/auditor) project will regularly evaluate and update the ArcGIS Online item's information (tags, thumbnail, title, etc) based off of information from a separate table and from the SGID. It will also update the ArcGIS Online item's metadata from the SGID's geodatabase.
 
-If the data is shared with the [SGID Open Data](https://opendata.gis.utah.gov/), it will have the enable multiple formats download option selected to allow for File Geodatabases and other non default downloads.
+If the dataset is shared with our [Open Data site](https://opendata.gis.utah.gov/), we will enable the multiple formats download option in the ArcGIS Online item page to allow users to download File Geodatabases and other non default download options.
 
-The [minimum metadata content]({% link about/policy/metadata/index.md %}) will be populated. When creating new metadata, it is recommended that you use the [metadata form](https://docs.google.com/forms/d/1u7gwdmRN-83Kh5zizi-kHRObeoNyaakM3scPkLT3zKY/edit).
+We will populate the [minimum metadata content]({% link about/policy/metadata/index.md %}) in the SGID geodatabase. We recommended that you use the [metadata form](https://docs.google.com/forms/d/1u7gwdmRN-83Kh5zizi-kHRObeoNyaakM3scPkLT3zKY/edit) to create new metadata for the SGID.
 
-All empty geometries and duplicate records will be removed, with [Sweeper](https://github.com/agrc/sweeper) or some other form of automation.
+The domain values for any new database domains must match domain their descriptions. See the [domain document](https://docs.google.com/document/d/12bdtmtv-ZVF9g-lFshbobx8DbKX9LfbL44HawLjFTVw/edit?usp=sharing) for more information on this effort.
 
-All new database domains will comply with the rule stating, domain values must match domain descriptions. See the [domain document](https://docs.google.com/document/d/12bdtmtv-ZVF9g-lFshbobx8DbKX9LfbL44HawLjFTVw/edit?usp=sharing) for more information on this effort.
+We will use [Sweeper](https://github.com/agrc/sweeper) (or some other form of automation) to remove all empty geometries and duplicate records.
 
-The ESRI default Resolution and Tolerance will be used.
+The SGID uses the ESRI default Resolution and Tolerance:
 
 - XY Resolution: 0.0001 Meter
 - XY Tolerance: 0.001 Meter
 
+### OpenSGID
```

Comment:
```suggestion
### Open SGID
```

Reply:
I have always used OpenSGID as a one-word brand name. Should I always be separating them into two words?

Reply:
yeah, i think two words is 👍🏼